Documentation fixes (#302)
MicrosoftDocs/microsoft-authentication-library-dotnet: msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md

- On Mac, Linux, and versions of Windows earlier than 10 or Windows Server 2019, MSAL will fall back to a browser.
- Updated WAM broker is not available on UWP due to Windows API limitations. UWP apps will use the legacy WAM implementation.

## Package availability

> To use the new broker, developers will need to use the <xref:Microsoft.Identity.Client.Broker.BrokerExtension> class, hosted in the <xref:Microsoft.Identity.Client.Broker> package. Most of the .NET platform variants supported by MSAL.NET will need that package only, with a few exceptions. See the table below for a detailed mapping.

Reviewer: ```suggestion
To use the new broker, developers will need to call <xref:Microsoft.Identity.Client.Broker.BrokerExtension.WithBroker(Microsoft.Identity.Client.PublicClientApplicationBuilder,Microsoft.Identity.Client.BrokerOptions)>, hosted in the <xref:Microsoft.Identity.Client.Broker> package. Most of the .NET platform variants supported by MSAL.NET will need that package only, with a few exceptions. See the table below for a detailed mapping.
```

> To use the new broker, developers will need to use the <xref:Microsoft.Identity.Client.Broker.BrokerExtension> class, hosted in the <xref:Microsoft.Identity.Client.Broker> package. Most of the .NET platform variants supported by MSAL.NET will need that package only, with a few exceptions. See the table below for a detailed mapping.

Reviewer 2: ```suggestion
To use the broker, developers will need to use the <xref:Microsoft.Identity.Client.Broker.BrokerExtension> class, hosted in the <xref:Microsoft.Identity.Client.Broker> package. Most of the .NET platform variants supported by MSAL.NET will need that package only, with a few exceptions. See the table below for a detailed mapping.
```

| Framework                       | [Microsoft.Identity.Client](https://www.nuget.org/packages/Microsoft.Identity.Client/) | [Microsoft.Identity.Client.Broker](https://www.nuget.org/packages/Microsoft.Identity.Client.Broker/) | [Microsoft.Identity.Client.Desktop](https://www.nuget.org/packages/Microsoft.Identity.Client.Desktop/) |
|:--------------------------------|:--------------------------|:---------------------------------|:----------------------------------|
| net48                           | ✅                       | ✅                               | ✅ (not recommended)             |
| net6.0                          | ✅                       | ✅                               | ⛔                               |
| net6.0-windows                  | ✅                       | ⛔                               | ⛔                               |

> | UWP                             | ✅                       | ⛔                               | ⛔                               |

Reviewer 2: ```suggestion
```

> | net48                           | ✅                       | ✅                               | ✅ (not recommended)             |
> | net6.0                          | ✅                       | ✅                               | ⛔                               |
> | net6.0-windows                  | ✅                       | ⛔                               | ⛔                               |
> | UWP                             | ✅                       | ⛔                               | ⛔                               |
> | .NET MAUI                       | ✅                       | ⛔                               | ⛔                               |
> | netcoreapp3.1 (not recommended) | ✅                       | ✅                               |✅ (not recommended)              |

Reviewer: Having checkmarks, for ex, on net6.0 and Microsoft.Identity.Client, makes it seem like you can use just that package, when you can't. So maybe this is clearer that you need to use MSAL.Broker for those platforms? Or maybe leave the checkmarks but add asterisk on MSAL.Broker and MSAL.Desktop column names that they have to be used in addition to MSAL package.

```suggestion
| net48                           | ⛔                       | ✅                               | ✅ (not recommended)             |
| net6.0                          | ⛔                       | ✅                               | ⛔                               |
| net6.0-windows                  | ✅                       | ⛔                               | ⛔                               |
| UWP                             | ✅                       | ⛔                               | ⛔                               |
| .NET MAUI                       | ✅                       | ⛔                               | ⛔                               |
| netcoreapp3.1 (not recommended) | ⛔                       | ✅                               |✅ (not recommended)              |
```

> | netcoreapp3.1 (not recommended) | ✅                       | ✅                               |✅ (not recommended)              |

Reviewer 2: ```suggestion
```

> Because the `netcoreapp3.1` target framework has reached the end of life in December 2022, our team will no longer offer support for applications using MSAL with the specific version of .NET Core. We recommend upgrading to the latest version where possible.

Reviewer: ```suggestion
Because the `netcoreapp3.1` target framework has reached the end of life in December 2022, our team will no longer offer support for applications using MSAL with this specific version of .NET Core. We recommend upgrading to the latest version where possible.
```

> Because the `netcoreapp3.1` target framework has reached the end of life in December 2022, our team will no longer offer support for applications using MSAL with the specific version of .NET Core. We recommend upgrading to the latest version where possible.

Reviewer 2: ```suggestion
```

Reviewer 2: No reason to talk about historical events :)
